ml-agents/setup.py

```diff
@@ -23,7 +23,11 @@
         "License :: OSI Approved :: Apache Software License",
         "Programming Language :: Python :: 3.6",
     ],
-    packages=["mlagents.trainers", "mlagents.trainers.bc", "mlagents.trainers.ppo"],  # Required
+    packages=[
+        "mlagents.trainers",
+        "mlagents.trainers.bc",
+        "mlagents.trainers.ppo",
+    ],  # Required
```

Comment:
Thanks~

Reply:
👍

Reply:
I think @ervteng's solution is better though.